src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person
+}
+
+export const PersonLink: FC<Props> = ({ person }) => (
+  <Link
+    className={cn({
+      'has-text-danger': person.sex === 'f',
```

Comment:
It would be cool not to use magic values and put such strings into constants and reuse them:
```
const FEMALE = 'f';
const NOT_SET_VALUE = '-';
```
This will increase the readability of the code and save us from possible errors.
This applies to development in general.